### DISCUSSION TITLE
When the Image.IdentityAsync(Stream) return null value?

### DISCUSSION THREAD

**Discussion by ntruongvux** [SixLabors/ImageSharp#1685] [Q&A] [answered]
When I use IdentityAsync to get image information but it always returns null.

**Comment by JimBobSquarePants** [comment]
You’ll need to provide a lot more information than that for anyone to be able to help in any way. At an absolute minimum a code sample and image.

**Reply by ntruongvux** [reply]
I use the image below to save to AzureBlob and read it to MemoryStream.
Can you provider the case can return null of the Image.IndentifyAsync(Stream stream);

![mountain](https://user-images.githubusercontent.com/55976935/124340922-fede3780-dbe2-11eb-9b6b-c69312344a1a.jpg)

**Comment by dlemstra** [comment, marked as answer]
You are getting null returned because that image is an unsupported format. The extension is `.jpg` but it actually is a `.webp` file that is currently not supported by this library.

**Reply by ntruongvux** [reply]
Yeah, Thank you so much
I don't know it's `.webp` 
But how to check the real extension of the image?

**Reply by dlemstra** [reply]
You are sort of doing that now 😉. You could also use `IdentifyWithFormatAsync` and check the format that is being returned.

**Reply by ntruongvux** [reply]
Thanks to @dlemstra
